dev/tasks/linux-packages/github.linux.amd64.yml

```diff
@@ -73,7 +73,6 @@ jobs:
           sudo apt update
           sudo apt install -y \
             apt-utils \
-            createrepo \
```

Comment:
@kou apt failed to locate createrepo on the graviton machines despite the same ubuntu
looked for its usage and seems like we're using it only from the binary release scripts, so I removed it

Reply:
It's used to test RPM packages.
Could you revert this?

Reply:
Resolved